src/plugins/discover/public/application/context/services/context.ts

```diff
@@ -29,7 +29,7 @@ const LOOKUP_OFFSETS = [0, 1, 7, 30, 365, 10000].map((days) => days * DAY_MILLIS
  * Fetch successor or predecessor documents of a given anchor document
  *
  * @param {SurrDocType} type - `successors` or `predecessors`
- * @param {IndexPattern} indexPattern
+ * @param {DataView} indexPattern
```

Comment:
Here should be `dataView` instead of `indexPattern`.

Reply:
yes, focused on types in this PR but there is plenty of more to clean up and rename, let's do it step by step